components/requests/PersonalInformationCard.tsx

```diff
@@ -1,10 +1,12 @@
 import { Box, HStack, VStack, Text, Divider, Badge, Link } from '@chakra-ui/react'; // Chakra UI
 import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card component
-import { Applicant } from '@lib/graphql/types'; // Applicant type
+// TODO: Temporary fix - add back in API hookup
+// import { Applicant } from '@lib/graphql/types'; // Applicant type
 import { MouseEventHandler } from 'react'; // React
 
 type PersonalInformationProps = {
-  readonly applicant: Applicant;
+  // TODO: Temporary fix - will be addressed in API hookup
+  readonly applicant: any;
```

Comment:
just curious why is this broken after the foreign key change

Is it just bc of the mock data?

Reply:
Good question! Yeah, it's because the mock data no longer fits the `Applicant` type. Adding the foreign keys to the `applicant` model makes the `Applicant` type also contain the nested `guardian` and `medicalInformation` objects - the workaround is to use TypeScript's `Pick` type to just include the fields that are applicable to the View Request page. cc @jeffreyzhang2001 who will address this in Request page API hookup.